internal/lsp/examples/index.json

```diff
@@ -1,22 +0,0 @@
-{
```

Comment:
This file is automatically updated by this workflow: https://github.com/StyraInc/regal/blob/main/.github/workflows/update-example-index.yaml so that when new docs are added, we link to them automatically from Regal.

Reply:
Ah, I had missed that! Will revert and switch to use a load once function. Thanks!